Scientific_workflows/WIT_Metrics/wit_metrics.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<p>Can you please provide an example ID, shape, and WIT file for running on Sandbox?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
```
fid_file = "/g/data/u46/users/ea6141/wlinsight/Barmah_Ramsar_ANAE_SYSID.csv"
shapefile = '/g/data/r78/DEA_Wetlands/shapefiles/MDB_ANAE_Aug2017_modified_2019_SB_3577.shp'
wit_data_file = '/g/data/u46/users/ea6141/wlinsight/Barmah_Ramsar_wit.csv
```
Shall I say uploading those files to sandbox and then the thing should run?

Reply:
I said.

Reply:
How large are these files? Maybe we can put them in dea-notebooks... otherwise maybe there's a small polygon we could use as an example

Reply:
The one in the notebook is very large. We could use a smaller shape file for an example. Work on it.

Reply:
Done. data uploaded in `example_data`